docker/Dockerfile.cpu

```diff
@@ -0,0 +1,28 @@
+FROM pytorch/pytorch:2.1.0-cuda11.8-cudnn8-runtime
+
+# Set the working directory in the container
+WORKDIR /usr/src/app
+
+# Install build tools and compilers
+# Avoid user interaction with tzdata when installing
+ENV DEBIAN_FRONTEND=noninteractive
+
+# Install build tools and compilers
+RUN apt-get update && apt-get install -y \
+  build-essential \
+  cmake \
+  && rm -rf /var/lib/apt/lists/*
+
+# Copy the current directory contents into the container at /usr/src/app
+COPY . .
```

Comment:
Dont copy everything
this will copy a lot of unnecessary items and files. Please copy only the required folders and files

Additioanlly when please copy .env.auto -> .env on the docker

Reply:
Same goes for all the other dockerfiles

Reply:
regarding the copy we update the .dockerignore to ignore all the unnecessary files,
fixed/update the .env for all

Reply:
not a good approach. some builds might require less or more files. remove dockerignore completly

Reply:
ok, what folder files do you wish to copy via docker and are there any differences for each build auto/cpu/gpu12/11?

Reply:
fixed